packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -116,6 +161,7 @@ export const prepareStakingAccountKit = (baggage, makeRecorderKit, zcf) => {
               Any.toJSON(
                 MsgDelegate.toProtoMsg({
                   delegatorAddress,
+                  // @ts-expect-error Type 'CosmosValidatorAddress' is not assignable to type 'string'.
                   validatorAddress,
```

Comment:
to get from a `ChainAddress` to a string, pick out the `.address`:

```suggestion
                  validatorAddress: validatorAddress.address,
```

... which suggests that `validatorAddress` is a poor name for the parameter.
`types.d.ts` calls it `validator`.

Reply:
This works for `ChainAddress['address']` which is a string. `CosmosValidatorAddress['address']` is a template string: `${string}valoper${string}`